Pipfile

```diff
@@ -18,3 +18,5 @@ psycopg2 = "~=2.8.5"
 scipy = "~=1.5"
 tqdm = "==4.29.1"
 requests = "~=2.25"
+fs = "*"
+"fs.sshfs" = "*"
```

Comment:
Just curious why we have double quotes here. Is this file generated automatically via `pipenv`?

Reply:
Yeah it was added automatically, I believe using `pipenv install fs.sshfs`.